src/containers/App.js

```diff
@@ -198,6 +199,13 @@ class App extends Component {
                 path={`/user/:userId/:listName(${listNames})/:mediaType(all|movie|tv)`}
                 component={UserList}
               />
+              <Route
+                exact
+                path="/forgot_password"
+                render={() => (
+                  <NewPassword />
```

Comment:
If `NewPassword` doesn't take any props, you can write:
```js
<Route
  ...
  component={NewPassword}
/>
```
instead of using render